src/core/plugins/json-schema-5-samples/fn/index.js

```diff
@@ -635,3 +635,7 @@ const resolver = (arg1, arg2, arg3) => [arg1, JSON.stringify(arg2), JSON.stringi
 export const memoizedCreateXMLExample = memoizeN(createXMLExample, resolver)
 
 export const memoizedSampleFromSchema = memoizeN(sampleFromSchema, resolver)
+
+export const getSchemaObjectTypeLabel = (label) => label
```

Comment:
The input is `type` not a `label`.

```suggestion
export const getSchemaObjectTypeLabel = (type) => type
```

But anyway we can make this point-free by:

```js
import identity from "lodash/identity"

```suggestion
export const getSchemaObjectTypeLabel = identity
```